from tensorflow.keras.models import Sequential
from tensorflow.keras.layers import Convolution2D,MaxPooling2D,Flatten,Dense
from tensorflow.keras.preprocessing.image import ImageDataGenerator as idm
import numpy as np
import warnings
#Supressing warnings
warnings.filterwarnings('ignore')
from google.colab import drive
drive.mount('/content/drive')
Mounted at /content/drive
# Creating augmentation on training variable
train_flowers=idm(rescale=1./255,zoom_range=0.2,horizontal_flip=True)

# Passing training data to train variable
Xtrain = train_flowers.flow_from_directory('/content/123128873_546b8b7355_n.jpg',target_size=(76,76),class_mode='categorical',batch_size=100)
  File "<ipython-input-12-00a25734f87d>", line 5
    Xtrain = train_flowers.flow_from_directory('/content/Flowers-Dataset (2).zip'/,target_size=(76,76),class_mode='categorical',batch_size=100)
                                                                                  ^
SyntaxError: invalid syntax
# Creating augmentation on testing variable
test_flowers=idm(rescale=1./255)

# Passing testing data to test variable
Xtest = test_flowers.flow_from_directory('/content/drive/MyDrive/flower/Test',target_size=(76,76),class_mode='categorical',batch_size=100)
Flower_model = Sequential()
Flower_model.add(Convolution2D(32,(3,3),activation='relu',input_shape=(76,76,3)))
Flower_model.add(MaxPooling2D(pool_size=(2,2)))
Flower_model.add(Flatten())
Flower_model.add(Dense(300,activation='relu'))
Flower_model.add(Dense(150,activation='relu'))
Flower_model.add(Dense(5,activation='softmax'))
Flower_model.compile(optimizer='adam',loss='categorical_crossentropy',metrics=['accuracy'])
Flower_model.fit_generator(Xtrain,steps_per_epoch= len (Xtrain),epochs= 8,validation_data=Xtest,validation_steps= len (Xtest))
Flower_model.save('Flower.h5')
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-8-1569fa3f58b3> in <module>
----> 1 Flower_model.save('Flower.h5')

NameError: name 'Flower_model' is not defined
from tensorflow.keras.preprocessing import image
test_img=image.load_img('/content/8181477_8cb77d2e0f_n.jpg/content/drive/MyDrive/flower/Train/rose/8181477_8cb77d2e0f_n.jpg',target_size=(76,76))
test_img
---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-7-30ddc2a48397> in <module>
----> 1 test_img=image.load_img('/content/8181477_8cb77d2e0f_n.jpg/content/drive/MyDrive/flower/Train/rose/8181477_8cb77d2e0f_n.jpg',target_size=(76,76))
      2 test_img

NameError: name 'image' is not defined